.github/workflows/build.yml

```diff
@@ -180,3 +180,17 @@ jobs:
     - name: test
       shell: bash
       run: ./ci/test.sh
+
+  testmacos1015:
+    runs-on: macos-10.15
+    steps:
+    - uses: actions/checkout@v2
+    - name: install
+      shell: bash
+      run: ./ci/install_macos.sh
+    - name: build
+      shell: bash
+      run: ./ci/build.sh clang++ Release 7 ON SSE42 -DPython_ROOT_DIR=/usr/local/opt/python@3.8
```

Comment:
Annoying brew python installs 3.7 and boost_python install 3.8... it's _probably_ valid for to built against slightly different minor versions (https://stackoverflow.com/questions/28830653/build-boost-with-multiple-python-versions) but this is more correct. Also there's issues in the boost-python CMake which make this trickier. Yeah...

Reply:
Done